### ISSUE TITLE
Force-related issues when fine-tuning the model

### ISSUE THREAD

**Issue by linanali0416** [ACEsuit/mace#1481]
<img width="1518" height="917" alt="Image" src="https://github.com/user-attachments/assets/f781e103-567e-4b04-8000-09998a145f41" />
When fine-tuning the MPA model, why does the situation shown in the fourth figure occur, with many scatter points lying outside the diagonal? It looks like the MACE_FORCES of these structures are zero.

**Comment by gabor1** [comment]
You've correctly identified that there is a subpopulation of test and train points with zero MACE forces. look at those structures manually, what is different about them? MACE can only produce a zero force if an atom is isolated or if there is high symmetry. in both cases reference forces should also be zero, which they are not here. So something is fishy with these structures or data. 

I also see 4 more suspicious points, this is where the MACE forces are opposite sign to DFT forces, with forces being large. this is pretty rate, so I would add those to the list of suspicious configurations. 

Can you also please verify that your forces and stresses are really the negative gradients of the energy, i.e. with the correct sign?

**Comment by ilyes319** [comment]
Hey @linanali0416,

Can you please share the log files of your finetuning?
I would look into these structures very carefully. Usually these are data problems, for example unconverged DFT. The way to do that is first visual inspection. Look at if you have high symmetries which is the only way mace could produce zero forces.

Can you summarize how you generated your data?

**Comment by linanali0416** [comment]
> Hey [@linanali0416](https://github.com/linanali0416),
> 
> Can you please share the log files of your finetuning? I would look into these structures very carefully. Usually these are data problems, for example unconverged DFT. The way to do that is first visual inspection. Look at if you have high symmetries which is the only way mace could produce zero 
> Can you summarize how you generated your data?

[MACE_FT_run-123_debug.log](https://github.com/user-attachments/files/28060077/MACE_FT_run-123_debug.log)

Thank you very much for your reply! The log file has been uploaded. The target dataset was obtained in two ways. The first is REICO sampling, where structural optimization is performed first, and then the trajectory structures are extracted for single-point energy calculations. The second way is to perform structural optimization on some stable structures I need to calculate, and then perform single-point calculations on their trajectories. And the converged structures were extracted using the vaspxyz.py script.

[vaspxyz.txt](https://github.com/user-attachments/files/28060362/vaspxyz.txt)

**Comment by gabor1** [comment]
Please do the other things we've asked, ie. identify the group of structures that have zero mace force and nonzero reference force and figure out what's special about them

**Comment by linanali0416** [comment]
> Please do the other things we've asked, ie. identify the group of structures that have zero mace force and nonzero reference force and figure out what's special about them

[DFT_valid.txt](https://github.com/user-attachments/files/28135652/DFT_valid.txt)

[MACE_valid.txt](https://github.com/user-attachments/files/28135657/MACE_valid.txt)
These are the structure files corresponding to DFT_valid and MACE_valid. Could you please take a look? I've noticed that there are many structures with large force deviations. For example, when two H atoms form H₂ in a structure, the force component deviations of the two H atoms are very large; besides that, other atoms also exhibit force deviations. I'm not sure how to handle these structures with large force deviations. Should I delete them and retrain the model, or should I perform structural optimization or single-point calculations on them and then add them to the dataset?Thanks
